pkgs/top-level/all-packages.nix

```diff
@@ -6165,6 +6165,8 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) AppKit Security;
   };
 
+  synthesia = callPackage ../games/synthesia { };
```

Comment:
Please move the package to `pkgs/by-name/sy/synthesia/package.nix` and remove this line, thanks!